public/locales/en/translation.json

```diff
@@ -295,6 +295,15 @@
           "workflowVariable": "Workflow Variable"
         }
       },
+      "deleteNamespace": {
+        "title": "Delete Namespace",
+        "description": "This will permanently delete the current active namespace and all resources associated with it.",
+        "deleteBtn": "Delete Namespace",
```

Comment:
```suggestion
        "title": "Delete namespace",
        "description": "This will permanently delete the current active namespace and all resources associated with it.",
        "deleteBtn": "Delete namespace",
```